packages/react-sdk/src/index.tsx

```diff
@@ -253,5 +252,5 @@ export function useRequestFeedback() {
 export function useSendFeedback() {
   const { client } = useContext<ProviderContextType>(ProviderContext);
   return (opts: Omit<Feedback, "userId" | "companyId">) =>
-    client?.feedback(opts);
+    client?.feedback(opts as Feedback);
```

Comment:
can we align these so we don't have to cast it? feels like a code smell

Reply:
Added a new type, the `Omit` is not loving me